ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/DefendantMapperTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.cmc.ccd.mapper.defendant;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.gov.hmcts.cmc.ccd.config.CCDAdapterConfig;
+import uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleTheirDetails;
+
+@SpringBootTest
+@ContextConfiguration(classes = CCDAdapterConfig.class)
+@RunWith(SpringJUnit4ClassRunner.class)
+public class DefendantMapperTest {
+
+    @Autowired
+    private DefendantMapper mapper;
+
+    @Test(expected = NullPointerException.class)
+    public void mapToShouldThrowExceptionWhenTheirDetailsIsNull() {
+        mapper.to(null, SampleClaim.getDefault());
```

Comment:
why are you not checking the exception message here ? where is the assertion for that ?

Reply:
No requirement to expect a certain message, and no business logic is based on those message, thus didn't need it.